ratpack-core/src/main/java/ratpack/exec/internal/DefaultExecController.java

```diff
@@ -160,7 +160,7 @@ public ExecStarter eventLoop(EventLoop eventLoop) {
 
       @Override
       public ExecStarter onError(Action<? super Throwable> onError) {
-        this.onError = onError;
+        this.errorListeners.add((e, t) -> onError.execute(t));
```

Comment:
with this, multiple `onError` calls can be made and they will all be fired on an error. This may constitute a breaking behavior change.

Reply:
@danveloper why do we need to have this multiple error handler behaviour?